public/css/PMR.css

```diff
@@ -1,24 +1,42 @@
- /*subject to change*/
-input[type=text], select {  
-    font-family: "Lucida Console"; 
-    font-size: 15px;
-    width: 50%;
-    padding: 15px 30px 15px; /*top, right, bottom, left */
-    margin: 10px 0px;
-    display: inline-block;
-    border: 1px solid #ccc;
-    border-radius: 4px;
-    box-sizing: border-box;
+.dropdown{	//dropdown button
+    font-family: "Lucida Console";
+    font-size: 10px;
+    margin: 10px 0;
+    padding: 15px 30px;
 }
 
-.chart-container {       /*Update colour scheme*/
-  font: 10px "Lucida Console";
-  background-image: linear-gradient(180deg, #ffffff, #99ccff); /*gradient?/ different for each*/
-  border-radius: 0.5em;
-  margin-top: 2em;
-  height: 500px;
+.chart-container {	
+    background-color: #eceff1;
```

Comment:
If this is left unspecified, it will inherit the `background-color` from `body` (which is probably what you wanted).